Cargo.toml

```diff
@@ -19,17 +19,17 @@ dirs = "3.0.1"
 tokio-socks = "0.5"
 clap = { version = "3.2.22", features = ["derive"] }
 bitcoind = "0.36"
-libtor = {version = "47.13.0", optional = true}
-mitosis = {version = "0.1.1", optional = true}
+libtor = { version = "47.13.0", optional = true }
+mitosis = { version = "0.1.1", optional = true }
 log4rs = "1.3.0"
-
+openssl-sys = { version = "0.9.68", optional = true }
 
 #Empty default feature set, (helpful to generalise in github actions)
 [features]
 default = ['tor']
+vendored-openssl = ["openssl-sys/vendored"]
```

Comment:
As discussed in the dev call, let's change this to `build-mac-os`, that's a more clear name. 

Add a comment above as a todo for adding more platform-specific build support.

We can create two different follow-up issues on of this.

- Short term: Add more platform specific fix so more devs can run it wherever they need to.
- Long term: Create a docker based solution that can run on any platform.
- Even Longer term: Ensure **Reproducible Builds** for all the apps and deamons.

For now, I am inclined to merge this as a short-term fix for #127 .